### DISCUSSION TITLE
Next.JS always removing query params on the URL if not use trailing slash

### DISCUSSION THREAD

**Discussion by randidev** [vercel/community#885] [Help]
Hello!
So I have my Next.JS app with static HTML export, the issue is whenever I type some query params on the URL it will be removed immediately if the query params not with the trailing slash. for example:
* When the URL is like this `some-page?test=123` it will removed the query params and redirecting to `some-page/`
* It works if the URL is like this `some-page/?test=123`

Is there any way to make the first one works?

**Comment by amyegan** [comment]
Hi @randidev! Can you share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) (or your repo if it is public) to help us investigate the problem?
